src/renderers/WebGLCubeRenderTarget.js

```diff
@@ -45,46 +45,33 @@ WebGLCubeRenderTarget.prototype.fromEquirectangularTexture = function ( renderer
 			tEquirect: { value: null },
 		},
 
-		vertexShader: [
+		vertexShader: /* glsl */`
+varying vec3 vWorldDirection;
 
-			"varying vec3 vWorldDirection;",
```

Comment:
The existing formatting in `WebGLCubeRenderTarget` is actually more readable than the new one. Probably better to align `PMREMGenerator` to `WebGLCubeRenderTarget`.

Reply:
Maybe we need a GLSL code formatter.

Reply:
Please no over-engineering 😉